pkg/models/system_intake_grb_reviewers.go

```diff
@@ -0,0 +1,45 @@
+package models
+
+import (
+	"github.com/google/uuid"
+)
+
+type SIGRBReviewerRole string
+
+const (
+	SIGRBRRCoChairCIO          SIGRBReviewerRole = "CO_CHAIR_CIO"
+	SIGRBRRCoChairCFO          SIGRBReviewerRole = "CO_CHAIR_CFO"
+	SIGRBRRCoChairHCA          SIGRBReviewerRole = "CO_CHAIR_HCA"
+	SIGRBRRACA3021Rep          SIGRBReviewerRole = "ACA_3021_REP"
+	SIGRBRRCCIIORep            SIGRBReviewerRole = "CCIIO_REP"
+	SIGRBRRProgOpBDGChair      SIGRBReviewerRole = "PROGRAM_OPERATIONS_BDG_CHAIR"
+	SIGRBRRCMCSRep             SIGRBReviewerRole = "CMCS_REP"
+	SIGRBRRFedAdminBDGChair    SIGRBReviewerRole = "FED_ADMIN_BDG_CHAIR"
+	SIGRBRRProgIntBDGChair     SIGRBReviewerRole = "PROGRAM_INTEGRITY_BDG_CHAIR"
+	SIGRBRRQIORep              SIGRBReviewerRole = "QIO_REP"
+	SIGRBRRSubjectMatterExpert SIGRBReviewerRole = "SUBJECT_MATTER_EXPERT"
+	SIGRBRROther               SIGRBReviewerRole = "OTHER"
+)
+
+type SIGRBReviewerVotingRole string
+
+const (
+	SIGRBRVRVoting    SIGRBReviewerVotingRole = "VOTING"
+	SIGRBRVRNonVoting SIGRBReviewerVotingRole = "NON_VOTING"
+	SIGRBRVRAlternate SIGRBReviewerVotingRole = "ALTERNATE"
+)
+
+// SystemIntakeGRBReviewer describes
+type SystemIntakeGRBReviewer struct {
+	BaseStruct
```

Comment:
should we use the newer `BaseStructUser`?

Reply:
Added in [5b62e6f](https://github.com/CMSgov/easi-app/pull/2588/commits/5b62e6fd69099aff996fa9657b39d4988d6f0584)

Reply:
NIce! Thanks 😄